source/getting-started/index.markdown

```diff
@@ -9,30 +9,38 @@ sharing: true
 footer: true
 ---
 
-First you will need to install Home Assistant before we can get started:
-
-We have developed **Hass.io** to turn your device into a dedicated Home Assistant hub controlled by a nice webinterface.
-For Raspberry Pi and Intel NUC we offer configured images. Flash the image, setup WiFi (if required) and everything else can be done inside the Hass.io webinterface. Great, isn't it?
-
-Also Home Assistant runs on any other device that supports **Python 3**.
-<div class="text-center hass-option-cards" markdown="0">
-  <a class='option-card' href='/hassio/installation/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/home-assistant.png' />
-    </div>
-    <div class='title'>Install Home Assistant using Hass.io</div>
-  </a>
-  <a class='option-card' href='/docs/installation/virtualenv/'>
-    <div class='img-container'>
-      <img src='/images/supported_brands/python.svg' />
-    </div>
-    <div class='title'>Install Home Assistant on your computer</div>
-  </a>
-</div>
-<br>
-
-For alternative installation methods, please take a look at the [installation documentation](/docs/installation/).
-
-If you run into any issues, please see [the troubleshooting page](/docs/installation/troubleshooting/) or [communication channels](/help/). It contains solutions to many commonly encountered issues.
+The goal of this getting started guide is to install Hass.io on a Raspberry Pi 3. Hass.io is our own all in one solution that turns your Raspberry PI into the utlimate home automation hub.
+
+Follow this guide if you want to easily get started with Home Assistant or if you have none or little Linux experience. For advanced users, check our [alternative installation methods](/docs/installation/).
+
+### {% linkable_title Hardware requirements %}
+
+We will need a few things to get started with installing Home Assistant. Links below are linking to Amazon US. If you're not in the US, you should be able to find these items in web stores in your country.
+
+ - [Raspberry Pi 3 model B](http://a.co/gEfMqL4) + [Power Supply](http://a.co/cgKUgkt) (atleast 2.5A)
+ - [Micro SD Card](http://a.co/gslOydD). Get one that is Class 10 as they are more reliable. Size 32GB or bigger recommended.
+ - SD Card reader. Part of most laptops. Also available as [standalone USB sticks](http://a.co/5FCyb0N) (brand doesn't matter, just pick cheapest)
+ - Ethernet cable (optional, Hass.io can work with WiFi too)
+
+### {% linkable_title Software requirements %}
+
+ - Download [Hass.io image for Raspberry Pi 3][pi3]
+ - Download [Etcher] to write the image to an SD card
+
```

Comment:
Windows users should also download an editor like Notepad++to see correct line breaks in config files (to enable to Hass.io configurator in a later step)

Reply:
Good point, I'll add that.